### ISSUE TITLE
PyQT dependency issue

### ISSUE THREAD

**Issue by sdebionne** [conda-forge/silx-feedstock#41]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

I am trying to install `silx` in a new environment with:

```
mamba create -n silx -c conda-forge silx
````

but the packagge will not install because pyqt is missing

```
Encountered problems while solving:
  - nothing provides pyqt needed by silx-0.12.0-1
```

If you wonder, I get the same result with `conda` (vs `mamba`), only solving dependencies takes ages...

**Comment by sdebionne** [comment]
My bad, I forgot I was running on a ppc64le platform... pyqt needs to be installed with pip.

**Comment by t20100** [comment]
Latest version available in conda-forge is 0.15.2 (https://anaconda.org/conda-forge/silx).
Now, after 0.12.0, the conda package was splitted into silx and [silx-base](https://anaconda.org/conda-forge/silx-base).
And I think silx-base does not depend on pyqt.

**Comment by sdebionne** [comment]
I forgot about #37. So far only `silx-base` is published for `ppc64le` but not `silx` -since pyqt is not yet available for `ppc64le`.

Here is the recipe for installing on a Power9, HTH:

```
mamba install numpy h5py matplotlib -c conda-forge
pip install PyQt5
pip install silx
```
